rocky/tools/models.py

```diff
@@ -170,6 +182,55 @@ class STATUSES(models.TextChoices):
     def blocked(self):
         return self.status == OrganizationMember.STATUSES.BLOCKED
 
+    @property
+    def is_admin(self):
+        return self.groups.filter(name=GROUP_ADMIN).exists()
+
+    @property
+    def is_redteam(self):
+        return self.groups.filter(name=GROUP_REDTEAM).exists()
+
+    @property
+    def is_client(self):
+        return self.groups.filter(name=GROUP_CLIENT).exists()
```

Comment:
I think these properties should be cached. Also, are they still necessary since with this approach we don't have to check for fixed groups anymore but permissions?

Reply:
Indeed caching here is a good idea. We use these props in just a few cases where we want to know if a user is in a particular group for redirecting purposes, mostly used in onboarding.

Reply:
Those checks should be changed, because we should not hardcode these groups and that means those groups don't have to exist.

Reply:
Here decided that we are going to fix this in another PR. We have to think about the design of how we are going to use these groups in the future.